src/main/java/com/mentoring/mju/v1/racing/view/InputView.java

```diff
@@ -1,10 +1,45 @@
 package com.mentoring.mju.v1.racing.view;
 
+import java.util.Arrays;
+import java.util.List;
 import java.util.Scanner;
 
 public class InputView {
 
     private static final Scanner input = new Scanner(System.in);
 
+    private List<String> carNames; //차 이름을 담을 list 선언
+    private int Chance; //몇 번 시도할 것인지 입력받는 변수 선언
```

Comment:
```suggestion
    private List<String> carNames;
    private int chance;
```

1. 주석 제거해서 올려주세요 !
2. 필드명은 카멜케이스 지켜주세요~